src/npc/npc/magician.c

```diff
@@ -15,16 +15,11 @@ void callback_magician(__attribute__((unused)) object_t *npc,
     scene_t *scene,
     __attribute__((unused)) window_t *win)
 {
-    static int countor = 0;
```

Comment:
Without that, it will spam the  (because 'a' is for dialog but also interraction) until the player move from the npc and pass through the dialogues